content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -228,6 +228,13 @@ and Ready or completely terminated prior to launching or terminating another
 Pod. This option only affects the behavior for scaling operations. Updates are not
 affected.
 
+#### Min Ready Seconds
```

Comment:
I'd suggest we spell this as "Minimum ready seconds".

Reply:
Done. PTAL